ENGAGEHF/ENGAGEHFStandard.swift

```diff
@@ -59,7 +59,7 @@ actor ENGAGEHFStandard: Standard, EnvironmentAccessible {
                         continue
                     }
 
-                    let document = collection.document(sample.id.uuidString)
```

Comment:
I'm assuming that SpeziHealthKit just wrapped the `uuid` with `id` to make it conform to `Identifiable`, so they return the same value, right?

Reply:
Yes, that's the same.